src/renderer/services/file-management-system/test/file-management-system.test.ts

```diff
@@ -476,6 +476,51 @@ describe("FileManagementSystem", () => {
       }
     );
 
+    [UploadStage.ADDING_CHUNKS, UploadStage.WAITING_FOR_FIRST_CHUNK].forEach(
+      (stage) => {
+        it(`resumes sending chunks for an upload. Said upload has some chunks uploaded already, and is in an active FSS status for stage ${stage}`, async () => {
+          // Arrange
+          const uploadId = "234124141";
+          const upload: UploadJob = {
+            ...mockJob,
+            serviceFields: {
+              files: [
+                {
+                  file: {
+                    fileType: "text",
+                    originalPath: testFilePath,
+                  },
+                },
+              ],
+              fssUploadId: uploadId,
+              fssUploadChunkSize: 13,
+              type: "upload",
+            },
+          };
+          const fssUpload: JSSJob = {
+            ...mockJob,
+            currentStage: stage,
+          };
+          jss.getJob.onFirstCall().resolves(upload);
+          fss.getStatus.resolves({
+            uploadStatus: UploadStatus.WORKING,
+            chunkStatuses: [ChunkStatus.COMPLETE, ChunkStatus.COMPLETE, null as any, ChunkStatus.COMPLETE, null as any, ChunkStatus.COMPLETE],
+          });
+          jss.getJob.onSecondCall().resolves(fssUpload);
+
+          // Act
+          await fms.retry("mockUploadId", noop);
+
+          // Assert
+          expect(jss.createJob.called).to.be.false;
+          const actualOffset = fileReader.read.args[0][4];
```

Comment:
I like that we are testing at the interface between the I/O library and the application! But `args[0][4]` is uninterpretable -- this at least deserves a comment

Reply:
I'm guessing that one of those other arguments to `fileReader.read` is the `testFilePath`? These assertions might actually be more readable if we also had an `expect(actualFilePath).to.equal(testFilePath)`, since it took me a bit of reading to figure out that that's the file we're making this assertion about

Reply:
Ah I see I missed the boat. Here's a ping @hughes036 in case this is relevant